src/components/visualizations/chart/test/chartOptionsBuilder.spec.ts

```diff
@@ -3527,6 +3574,36 @@ describe("chartOptionsBuilder", () => {
                 ];
                 expect(chartOptions.yAxes).toEqual(expectedAxis);
             });
+
+            it("should return number values in tooltip when point is on Second Axis", () => {
```

Comment:
I think `should... on secondary axis` is fine. We don't need to capitalize words.

Reply:
updated